a3fe/run/stage.py

```diff
@@ -175,6 +175,10 @@ def __init__(
                 self.lam_vals = lambda_values
             else:
                 self.lam_vals = self._get_lam_vals()
```

Comment:
Please modify this to completely remove any reference to "somd.cfg" which should only be written by the engine config before it is used.

Reply:
Please check why somd.cfg is appearing in the stage input directories - it shouldn't.